sphinx/source/concepts.rst

```diff
@@ -25,6 +25,13 @@ The Key-Value Store is a collection of maps (associating a key to a value) that
 
 .. note:: Since all nodes in the CCF network can read the content of private maps, it is up to the application logic to control the access to such maps. Since every application endpoint has access to the identity of the user triggering it, it is easy to restrict which maps (and entries in those maps) a user can read or write to.
 
+Consensus algorithms
```

Comment:
Please have a look at line 22, consensus protocols are already mentioned there. It might be worth while that Raft is the default and PBFT is still in development.

Reply:
Are you proposing mentioning that Raft is default and PBFT is under development there too, or deleting this section and only adding that note there?